demos/teststand/dg_teststand_new_jump.py

```diff
@@ -1,7 +1,3 @@
-## simple jump on impact test stand
```

Comment:
why did you remove these?

Reply:
This file is the same as dg_teststand_jump.py file that Avadesh implemented. I removed it in my other branch.